modules/SlacSimulator/manifest.yaml

```diff
@@ -0,0 +1,32 @@
+description: SIL Implementation of SLAC data link negotiation according to ISO15118-3.
+provides:
+  evse:
+    interface: slac
+    description: SLAC interface implementation for EVSE side
+    config:
+      evse_id:
```

Comment:
The config options for evse_id, nid should be removed, they are not used and not present in the real slac module. The number_of_sounds exists in the real slac module, but should be removed here as well I think as it is not used in this module.